EXOSIMS/Observatory/TwoStarShades_mission.py

```diff
@@ -0,0 +1,567 @@
+            a3 = np.cross(a1, a2)  # last part of the A basis vectors
+
+            # finding sign of angle
+            # The star angular separation can be negative
+            u2_Az = np.dot(a3, u_new.T)
+            sgn = np.sign(u2_Az)
+            sgn[np.where(sgn == 0)] = 1
+            sd = sgn * sd
+        return sd
+
+    def mass_dec(self, dF_lateral, t_int):
+        """Returns mass_used and deltaV
+
+        The values returned by this method are used to decrement spacecraft
+        mass for station-keeping.
+
+        Args:
+            dF_lateral (astropy.units.Quantity):
+                Lateral disturbance force in units of N
+            t_int (astropy.units.Quantity):
+                Integration time in units of day
+
+        Returns:
+            tuple:
+                intMdot (astropy.units.Quantity):
+                    Mass flow rate in units of kg/s
+                mass_used (astropy.units.Quantity):
+                    Mass used in station-keeping units of kg
+                deltaV (astropy.units.Quantity):
+                    Change in velocity required for station-keeping in units of km/s
+
+        """
+        if self.counter_1 == 0:
+
+            intMdot = (dF_lateral / self.skEff / const.g0 / self.skIsp).to("kg/s")
+            mass_used = (intMdot * t_int).to("kg")
+            deltaV = (dF_lateral / self.scMass[:, 0] * t_int).to("km/s")
+            self.counter_1 = self.counter_1 + 1
+            return intMdot, mass_used, deltaV
+
+        else:
+            intMdot = (dF_lateral / self.skEff / const.g0 / self.skIsp).to("kg/s")
+            mass_used = (intMdot * t_int).to("kg")
+            deltaV = (dF_lateral / self.scMass[:, 1] * t_int).to("km/s")
+            self.counter_1 = 0
+            return intMdot, mass_used, deltaV
+
+    def calculate_slewTimes(self, TL, old_sInd, sInds, sd, obsTimes, currentTime):
+        """Finds slew times and separation angles between target stars
+
+        This method determines the slew times of an occulter spacecraft needed
+        to transfer from one star's line of sight to all others in a given
+        target list.
+
+        Args:
+            TL (:ref:`TargetList`):
+                TargetList class object
+            old_sInd (int):
+                Integer index of the most recently observed star
+            sInds (~numpy.ndarray(int)):
+                Integer indices of the star of interest
+            sd (~astropy.units.Quantity):
+                Angular separation between stars in rad
+            obsTimes (~astropy.time.Time(~numpy.ndarray)):
+                Observation times for targets.
+            currentTime (astropy Time):
+                Current absolute mission time in MJD
+
+        Returns:
+            ~astropy.units.Quantity:
+                Time to transfer to new star line of sight in units of days
+        """
+        # where does the conversion happens to day units, and intTimes
```

Comment:
what is this comment?

Reply:
I probably added this comment to ask in meeting last year and forgot about it, deleted. Fixed